Compiler warnings with g++-4.8-2ubuntu1~14.04

Not critical, but documented here for completeness.

```
../benchmarks/../laz-perf/io.hpp:207:55: warning: cast from type ‘const char*’ to type ‘unsigned char*’ casts away qualifiers [-Wcast-qual]
                 version.major = *(unsigned char *)data++;
                                                       ^
../benchmarks/../laz-perf/io.hpp:208:55: warning: cast from type ‘const char*’ to type ‘unsigned char*’ casts away qualifiers [-Wcast-qual]
                 version.minor = *(unsigned char *)data++;
                                                       ^
```

--- a/laz-perf/io.hpp
+++ b/laz-perf/io.hpp
@@ -204,8 +204,8 @@ namespace laszip {
                 coder = le16toh(coder);
                 data += sizeof(coder);
 
-                version.major = *(unsigned char *)data++;
-                version.minor = *(unsigned char *)data++;
+                version.major = *(const unsigned char *)data++;
+                version.minor = *(const unsigned char *)data++;
 
                 memcpy(&version.revision, data, sizeof(version.revision));
                 version.revision = le16toh(version.revision);

